packages/openapi-generator/src/generator.spec.ts

```diff
@@ -17,27 +17,6 @@ describe('generator', () => {
     mock.restore();
   });
 
-  it('getInputFilePaths returns an array of all file paths, including subdirectories', async () => {
```

Comment:
What's the reason for removing this test? 🤔

Reply:
This was removed by mistake, have reverted it now. Thanks for pointing this out 👍